### PR TITLE
WRKLDS-950: must-gather: add --since and --since-time flags to proposal

### PR DESCRIPTION
**Pull request by flavianmissi**

### CODE REVIEW

File: enhancements/oc/must-gather.md [hunk 1]
```diff
@@ -110,9 +114,10 @@ If the `oc adm must-gather` tool's pod cannot be scheduled or run on the cluster
 
 ### Implementation Details/Notes/Constraints
 
-What are the caveats to the implementation? What are some important details that
-didn't come across above? Go in to as much detail as necessary here. This might
-be a good place to talk about core concepts and how they releate.
+During the implementation of `--since` and `--since-time`, `--until` and
+`--until-time` were also considered.
+Supporting `until` flags considerably increase the complexity of the task, so more
+research is required before pursuing it.
```
Comment (ingvagabund) [inline enhancements/oc/must-gather.md:120]: Worth mentioning the limitation of the rotated logs and linking https://github.com/openshift/oc/pull/1653.

Comment (flavianmissi) [reply enhancements/oc/must-gather.md:120]: Good idea, updated.

**Comment by flavianmissi** [comment]
@ingvagabund I've addressed your comment, can you take another look? Thanks!

**Review by soltysh — APPROVED** [review]
/approve

**Pull request merged**